python/cugraph-pyg/cugraph_pyg/sampler/cugraph_sampler.py

```diff
@@ -140,12 +142,10 @@ def __neighbor_sample(
             # FIXME support variable num neighbors per edge type
             num_neighbors = list(num_neighbors.values())[0]
 
-        # FIXME eventually get uniform neighbor sample to accept longs
         if backend == "torch" and not index.is_cuda:
             index = index.cuda()
 
-        # FIXME resolve the directed/undirected issue
-        G = self.__graph_store._subgraph([et[1] for et in edge_types])
+        G = self.__graph_store._subgraph(edge_types)
 
         index = cudf.from_dlpack(index.__dlpack__())
```

Comment:
I would suggest using the to_dlpack object here or using `cp.asarray(a)` . 

https://pytorch.org/docs/stable/dlpack.html#torch.utils.dlpack.to_dlpack

```suggestion
        index = cudf.from_dlpack(index.__dlpack__())
```

Reply:
This doesn't work for cudf

Reply:
resolved, all dlpack calls removed